### ISSUE TITLE
s/anyref/externref in repo description

### ISSUE THREAD

**Issue by nilslice** [WebAssembly/reference-types#139]
Not a big thing, but saw there is mention of `anyref` in this repository's description: 
![image](https://user-images.githubusercontent.com/7517515/137511817-eccb4b42-e742-498a-bfc5-0154e6b6f000.png)

Apologies if this is correct and I've misunderstood what I thought was a name change! If not, I figured it should be changed for consistency. 

As an aside, there are other mentions of `anyref` in the repo, in comments, code, and documentation. I'd be happy to scrub through and change these but want to be sure this is an actual name change and not just my imagination 😆

**Comment by rossberg** [comment]
@nilslice, yes, the name change happened. Thanks for pointing out the oversights. OTOH, this proposal has already been merged into the main spec, so that this repo is no longer active. Not sure it's worth cleaning up retroactively.

**Comment by nilslice** [comment]
Good point! Yea, it might not be worthwhile after all. I suppose the counter argument is only that I tend to still land on these repos after a Google search, and the stale terms may lead to confusion.

**Comment by rossberg** [comment]
I'm fine either way, will happily accept a PR.
